world_universities_and_domains.json

```diff
@@ -32181,9 +32181,9 @@
     {
         "alpha_two_code": "IQ",
         "country": "Iraq",
-        "domain": "iu.edu.iq",
-        "name": "Ishik University",
-        "web_page": "http://www.iu.edu.iq/"
+        "domain": "ishik.edu.iq",
+        "name": "Ishik University (Kurdistan Region)",
```

Comment:
Please only use legal name of the university. Location names should be provided separately.